front-end/src/components/analysis-page/AnalysisPage.jsx

```diff
@@ -69,12 +69,10 @@ class AnalysisPage extends Component {
         return (
             <div className="AnalysisPage bg-purple-100 flex h-screen" data-testid="analysis-page">
               <div className="flex gap-y-10 w-full flex-col h-full">
-                <div className="justify-center flex mt-12">
-                  <BaseButton
-                    click={this.openMainPage}
-                    text="Return to Main Page" 
-                    size="sm"
-                  />
+                <div className="justify-left flex cursor-pointer m-5 w-[5em]"
+                  onClick={this.openMainPage}
+                  data-testid="logo-button">
+                  <Logo />
                 </div>
```

Comment:
For a11y, this probably should not be a div so that screen readers know it is clickable. Can we make it a button instead?

Alternatively, we can give it role="button"
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

Reply:
Yeah I agree that this probably shouldn't just be a div but maybe something that indicates it's clickable

Reply:
^^^ I think giving it the button role with aria label works best.